packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -2807,19 +2807,33 @@ export const isGETDashboardsAgentStatusParams = ajv.compile<GETDashboardsAgentSt
 	GETLivechatAnalyticsDashboardsAgentStatusParamsSchema,
 );
 
-type PUTLivechatPriority = {
-	name: string;
-};
+type Without<T, U> = { [P in Exclude<keyof T, keyof U>]?: never };
+type XOR<T, U> = T | U extends object ? (Without<T, U> & U) | (Without<U, T> & T) : T | U;
 
+type PUTLivechatPriority = XOR<{ name: string }, { reset: boolean }>;
```

Comment:
🤔 do we need this type?

Reply:
yes, it's the only proper way i found to type these parameters.

Reply:
What's the problem with `{ name } | { reset }` and using type guards at the endpoint to know which of the types to use?

Reply:
the de-structuring  was the problem, as we talked.